### PR TITLE
Add animated plots for StaticGeodesicsPlotter.

### PR DESCRIPTION
**Pull request by Varunvaruns9**
Fixes parts of #36

@shreyasbapat 

Demo:
55973705-c9686f00-5ca3-11e9-8ffa-80df9e4d22eb

### CODE REVIEW

**Comment by Varunvaruns9** [comment]
Now that #157 has finally been merged, this PR can be completed. I will resolve the conflicts and push the changes shortly.

**Comment by shreyasbapat** [comment]
Correct! I very much look forward to this!

**Comment by Varunvaruns9** [comment]
@shreyasbapat Done :)

File: src/einsteinpy/tests/test_plotting/test_staticgeodesicplotter.py [hunk 1]
```diff
@@ -43,6 +43,20 @@ def test_plot_calls_plt_show(mock_show):
     assert cl._attractor_present
 
 
+@mock.patch("einsteinpy.plotting.geodesics_static.plt.show")
+def test_plot_calls_plot_show(mock_animated_plt_show):
```
Comment (shreyasbapat) [inline src/einsteinpy/tests/test_plotting/test_staticgeodesicplotter.py]: Not a good test! The test should check and assert values related to animate and not show :)

Comment (Varunvaruns9) [reply src/einsteinpy/tests/test_plotting/test_staticgeodesicplotter.py]: Yes, I initially added the `show` method to `animate` and changed it later as suggested in #177  but forgot changing the tests. I'll make the changes ASAP.

**Comment by shreyasbapat** [comment]
This will now need some significant change and a rebase :)

**Comment by Varunvaruns9** [comment]
This one has been fixed.

**Pull request merged**